sample-app/screens/HomeScreen.js

```diff
@@ -227,12 +344,30 @@ export default class HomeScreen extends React.Component<Props, State> {
               onPress={() => this.makeGooglePayPayment(false)}
             />
           )}
+          <View style={styles.spacing} />
+          {isIos ? (
+            <ApplePayButton
+              style={styles.payButtonStyle}
+              setThemeStyle={ButtonTheme.Dark}
+              onPayPress={() => this.makeAppleNativePayment() }
+            />
+          ) : (
+            <GooglePayButton
+              style={styles.payButtonStyle}
+              onPayPress={() => this.makeGoogleNativePayment() }
```

Comment:
Could this call the same method as the normal button above? `this.makeGooglePayPayment(false)`

Reply:
👍